src/cli/templates.hh

```diff
@@ -1542,13 +1539,10 @@ width: 1024
 # The icon to use for identifying your app on MacOS.
 # mac_icon:
 
-# TODO description & value
-# mac_sign:
-
-# TODO description & value
+# macOS codesign certificate name.
```

Comment:
can we have a description about what it should be? where does it come from? etc.

Reply:
or maybe this document doesnt need descriptions and the docs site should just be improved? we could maybe even kill all the comments and just link to the docs?